_data-prepper/pipelines/configuration/sources/kinesis.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: kinesis
+parent: Sources
+grand_parent: Pipelines
+nav_order: 3
+---
+
+# kinesis
+
+You can use `kinesis` source in Data Prepper to ingest records from one or more [Amazon Kinesis](https://aws.amazon.com/kinesis/) Data Streams. 
+
+## Usage
+
+The following example pipeline specifies Kinesis as a source. It ingests data from multiple Kinesis Data streams named `stream1` and `stream2`. It also indicates the `initial_position`, which tells the pipeline from where to start reading Kinesis stream records:
+
+```yaml
+version: "2"
+kinesis-pipeline:
+  source:
+    kinesis:
+      streams:
+        - stream_name: "stream1"
+          initial_position: "LATEST"
+        - stream_name: "stream2"
+          initial_position: "LATEST"
+      aws:
+        region: "us-west-2"
+        sts_role_arn: "arn:aws:iam::123456789012:role/my-iam-role"
+```
+
+## Configuration options
+
+The following configuration options are supported for the `kinesis` source.
+
+Option | Required | Type     | Description
+:--- |:---------|:---------| :---
+`aws` | Yes      | AWS      | The AWS configuration. See [aws](#aws) for more information.
+`acknowledgments` | No       | Boolean  | When `true`, enables `kinesis` source to receive [end-to-end acknowledgments]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#end-to-end-acknowledgments) when events are received by OpenSearch sinks.
+`streams` | Yes      | List     | Multiple Kinesis Data streams that Data Prepper `kinesis` source uses to read records. You can configure up to 4 streams. For more information about `streams` configuration options, see [Streams](#streams).
+`codec` | Yes      | Codec    | The [codec](#codec) to apply.
+`buffer_timeout` | No       | Duration | The amount of time allowed for writing events to the Data Prepper buffer before timeout occurs. Any events that the source cannot write to the buffer during the specified amount of time are discarded. Default is `1s`.
+`records_to_accumulate` | No       | Integer  | The number of messages that accumulate before being written to the buffer. Default is `100`.
+`consumer_strategy` | No       | String   | Consumer strategy to use for ingesting Kinesis data streams. Default is `fan-out`. However, `polling` can also be used. if `polling` is enabled, additional configuration for `polling` will need to be added.
+`polling` | No       | String   | if `consumer_strategy` is set to `polling`, this config will need to be added. Refer to [polling](#polling).
```

Comment:
The type here is `polling`. Please update the type column and have it link to the `polling` section below.